app/data/jsonSchemaForm/fundingParameterIAUiSchema.ts

```diff
@@ -72,6 +87,9 @@ export const fundingParameterIAUiSchema = {
       amount: {
         "ui:title": `Additional Funding Amount`,
         "ui:widget": "NumberWidget",
+        "ui:tooltip": {
+          text: "<div><ul><li>The amount of payment from Funding Source N other than CIF.</li><li>Typically found in the original proposal (solicitation folder of the LAN).</li></ul></div>",
+        },
```

Comment:
solicitation folder on the LAN? based on the Word document?!

Reply:
fixed! great catch